discourse-mixin/alerts/alerts.libsonnet

```diff
@@ -0,0 +1,45 @@
+{
+  prometheusAlerts+:: {
+    groups+: [
+      {
+        name: 'DiscourseAlerts',
+        rules: [
+          {
+            alert: 'DiscourseRequestsHigh5XXs',
```

Comment:
Mixtool throws alert-name-camelcase for this alert

Reply:
Had to update my mixtool and that caught it! Renamed it to `DiscourseRequestsHigh5xxErrors` 👍